model_cards/English/toxicity.md

```diff
@@ -0,0 +1,91 @@
+# Toxicity 
+Toxicity classifies rude, disrespectful, or unreasonable comment that is likely to make people leave a discussion. This model is a Convolutional Neural Network (CNN) trained with GloVe word embeddings. You can also train your own deep CNN for text classification on our public toxicity dataset, and explore our open-source model training tools to train your own models.

```

Comment:
Make the `open-source model training tools` a link?

Reply:
Good catch, done!